crates/bevy_utils/src/default.rs

```diff
@@ -0,0 +1,25 @@
+/// Used to make initializing structs with defaults easier:
+/// ```
+/// use bevy_utils::default;
+///
+/// #[derive(Default)]
+/// struct Foo {
+///   bar: usize,
+///   baz: usize,
+/// }
+///
+/// // Normally you would do this:
+/// let foo = Foo {
+///   bar: 10,
+///   ..Default::default()
+/// };
+///
+/// // But now you can do this:
+/// let foo = Foo {
+///   bar: 10,
+///   ..default()
+/// };
+/// ```
+pub fn default<T: Default>() -> T {
```

Comment:
```suggestion
#[inline]
#[must_use]
pub fn default<T: Default>() -> T {
```

Reply:
inlining makes a lot of sense, but id prefer to not force `must_use` on people.

Reply:
Fair, I can accept that judgement call.